site/content/release-notes.md

```diff
@@ -25,6 +25,13 @@ title = "Release Notes"
   context. The default implementation of the *connection events* still defaults
   to the logging producer.
 
+### Deprecations
+
+* The Kura protocol adapter is being deprecated with 0.8. It will still be part
+  Hono 0.8, but may be removed in a future version. Both Kura and Hono have been
```

Comment:
IMHO it would be very beneficial if you could indicate which versions this applies to, i.e. Kura >= 4 and Hono >= 0.8. WDYT?